modules/launcher/src/main/assembly/assembly.xml

```diff
@@ -56,6 +56,7 @@
             <includes>
                 <include>broker.sh</include>
                 <include>broker.bat</include>
+                <include>mb.sh</include>
```

Comment:
The name "mb" feel bit ambiguous to me. Shall we rename to something like `broker-admin.sh`?

Reply:
+1